### ISSUE TITLE
[tune] Verbosity specification ignored

### ISSUE THREAD

**Issue by fruffy** [ray-project/ray#3928]
### System information
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 18.04
- **Ray installed from (source or binary)**: nightly build
- **Ray version**: latest
- **Python version**: 2.7/3
- **Exact command to reproduce**:
`sudo python python/ray/tune/examples/tune_mnist_ray.py`

### Describe the problem

I am trying to get Ray to not spit out anything while running. For example: 
```
== Status ==
Using FIFO scheduling algorithm.
Resources requested: 6/8 CPUs, 0/0 GPUs
Memory usage on this node: 11.5/16.7 GB
Result logdir: /home/fruffy/ray_results/tune_mnist_test
```
should not appear.
However, it seems that in ray.tune
` print(runner.debug_string(max_debug=99999))`
is printed regardless of what I specify.
I tried
`'config': {'log_level': "ERROR"}`
`ray.init(logging_level=logging.ERROR)`
`run_experiments({'tune_mnist_test': mnist_spec}, verbose=False)`
 
None of it worked. Is there a way to disable it?

### Source code / logs
```
# !!! Example of using the ray.tune Python API !!!
if __name__ == '__main__':
    parser = argparse.ArgumentParser()
    parser.add_argument(
        '--smoke-test', action='store_true', help='Finish quickly for testing')
    args, _ = parser.parse_known_args()

    register_trainable('train_mnist', train)
    mnist_spec = {
        'run': 'train_mnist',
        'stop': {
            'mean_accuracy': 0.99,
            'time_total_s': 600,
        },
        'config': {
            'log_level': "ERROR",
            "num_workers": 1,
            "num_gpus": 0,
            'activation': grid_search(['relu', 'elu', 'tanh']),
            # You can pass any serializable object as well
            'foo': grid_search([np.array([1, 2]),
                                np.array([2, 3])]),
        },
    }

    if args.smoke_test:
        mnist_spec['stop']['training_iteration'] = 2

    ray.init(logging_level=logging.ERROR)
run_experiments({'tune_mnist_test': mnist_spec}, verbose=False)
```

**Comment by richardliaw** [comment]
Can you try the latest master https://ray.readthedocs.io/en/latest/installation.html ? You should be able to set verbosity to 0, which should produce no output.

**Comment by richardliaw** [comment]
(addressed in #3906)

**Comment by fruffy** [comment]
Just tried it, worked. Thanks!
